themes/nord-color-theme.json

```
@@ -897,6 +897,16 @@
         "foreground": "#8FBCBB"
       }
     },
+    {
+      "name": "[JavaScript] Parentheses in Template Strings",
+      "scope": [
+        "source.js meta.embedded.line meta.brace.square",
+        "source.js meta.embedded.line meta.brace.round"
```

Even though this is a extension specific addition, we should add this since [`mgmcdermott.vscode-language-babel`][e] is the most used extensions for _JavaScript_ syntax highlighting.

```suggestion
        "source.js meta.embedded.line meta.brace.square",
        "source.js meta.embedded.line meta.brace.round",
        /* Required for extension `mgmcdermott.vscode-language-babel`. */
        "source.js string.quoted.template meta.brace.square",
        "source.js string.quoted.template meta.brace.round",
```

[e]: https://marketplace.visualstudio.com/items?itemName=mgmcdermott.vscode-language-babel